qa/standalone/mon/misc.sh

```diff
@@ -265,6 +270,8 @@ function TEST_mon_features() {
     jq_success "$jqinput" "$jqfilter" || return 1
```

Comment:
http://pulpito.front.sepia.ceph.com/yuriw-2022-09-14_16:47:07-rados-wip-yuri4-testing-2022-09-14-0617-distro-default-smithi/7032918/  is related. We need to update this now - this was done is later commit for Quincy https://github.com/ceph/ceph/pull/39187